src/ng/location.js

```diff
@@ -30,13 +30,18 @@ function parseAbsoluteUrl(absoluteUrl, locationObj) {
   locationObj.$$port = toInt(parsedUrl.port) || DEFAULT_PORTS[parsedUrl.protocol] || null;
 }
 
+var DOUBLE_SLASH_REGEX = /\s*[\\/]{2,}/;
+function parseAppUrl(url, locationObj) {
 
-function parseAppUrl(relativeUrl, locationObj) {
-  var prefixed = (relativeUrl.charAt(0) !== '/');
+  if (url.search(DOUBLE_SLASH_REGEX) === 0) {
```

Comment:
<sub>You could also add `^` at the beginning of `DOUBLE_SLASH_REGEX` and use  `DOUBLE_SLASH_REGEX.test(url)`?</sub>